assets/src/blocks/Submenu/SubmenuItems.js

```diff
@@ -0,0 +1,32 @@
+export const SubmenuItems = ({ menuItems }) => {
+
+  const renderMenuItems = (items) => {
+    return items.map(({ anchor, text, style, shouldLink, children }) => (
+      <li key={anchor} className={`list-style-${style || 'none'} ${shouldLink ? "list-link" : "list-heading"}`}>
+        {shouldLink ?
+          <a
+            className="icon-link submenu-link"
+            href={`#${anchor}`}
+          >
+            {text}
```

Comment:
Same problem here as the other blocks about RichText automagically escaping things
![Screenshot from 2020-10-19 11-43-50](https://user-images.githubusercontent.com/617346/96429289-14503480-1201-11eb-9d27-5b8dba7aa1a3.png)
The only solution we have used so far is to use `dangerouslySetInnerHTML`, which doesn't sound super good, we might have to look for a better solution at some point

Reply:
I instead unescaped what is returned as the block content, that way we can normally pass it to render without having to call `dangerouslySetInnerHTML` https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/363/files#diff-24dc7e1c4c410e70f8e7e29e8f70b65ade34c0800fbcc8877034cb3d00fea05eR28